includes/abstracts/llms-abstract-admin-wizard.php

```diff
@@ -0,0 +1,390 @@
+
+		/**
+		 * Filter the Save button text for a given step in the setup wizard
+		 *
+		 * The dynamic portion of this hook, `$step`, refers to the slug of the current step.
+		 *
+		 * @since [version]
+		 *
+		 * @param string $text Button text string.
+		 */
+		return apply_filters( "llms_setup_wizard_get_{$step}_save_text", $this->get_steps()[ $step ]['save'] ?? '' );
+	}
+
+	/**
+	 * Get the text to display on the "skip" buttons
+	 *
+	 * @since [version]
+	 *
+	 * @param string $step Step to get text for.
+	 * @return string Translated text.
+	 */
+	private function get_skip_text( string $step ): string {
+
+		/**
+		 * Filter the skip button text for a given step in the setup wizard
+		 *
+		 * The dynamic portion of this hook, `$step`, refers to the slug of the current step.
+		 *
+		 * @since [version]
+		 *
+		 * @param string $text Button text string.
+		 */
+		return apply_filters( "llms_setup_wizard_get_{$step}_skip_text", $this->get_steps()[ $step ]['skip'] ?? '' );
+
+	}
+
+	/**
+	 * Get the URL to a step
+	 *
+	 * @since [version]
+	 *
+	 * @param string $step Step slug.
+	 * @return string
+	 */
+	private function get_step_url( string $step ): string {
+		return add_query_arg(
+			array(
+				'page' => 'llms-' . $this->type,
+				'step' => $step,
+			),
+			admin_url()
+		);
+	}
+
+	/**
+	 * Get an array of step slugs => titles
+	 *
+	 * @since [version]
+	 *
+	 * @return array
+	 */
+	public function get_steps(): array {
+
+		/**
+		 * Filter the steps included in the setup wizard
+		 *
+		 * @since [version]
+		 *
+		 * @param string[] $steps Array of setup wizard steps. The array key is the slug/id of the step and the array value
+		 *                        is the step's title displayed in the wizard's navigation.
+		 */
+		return apply_filters( 'llms_setup_wizard_steps', $this->steps );
+
+	}
+
+	/**
+	 * Output the HTML content of the setup page
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function output(): void {
+		$views_dir = trailingslashit( esc_attr( $this->views_dir ) );
+		$step_html = '';
+		$steps     = $this->get_steps();
+		$current   = $this->get_current_step() ?? 'intro';
+		$prev      = $this->get_prev_step();
+		$next      = $this->get_next_step();
+		$transient = get_transient( $this->transient_key ) ?? [];
+
+		if ( in_array( $current, array_keys( $steps ), true ) ) {
+
+			ob_start();
+			include $views_dir . 'step-' . $current . '.php';
+			$step_html = ob_get_clean();
+
+		}
+
+		/**
+		 * Filter the HTML of a step within the setup wizard.
+		 *
+		 * The dynamic portion of this hook, `$current`, refers to the slug of the current step.
+		 *
+		 * This filter can be used to output the HTML for a custom step in the setup wizard.
+		 *
+		 * @since [version]
+		 *
+		 * @param string $step_html HTML of the step.
+		 * @param LLMS_Admin_Setup_Wizard $wizard Setup wizard class instance.
+		 */
+		$step_html = apply_filters( "llms_setup_wizard_{$current}_html", $step_html, $this );
```

Comment:
```suggestion
		$step_html = apply_filters( "llms_{$this->type}_wizard_{$current}_html", $step_html, $this );
```
Also update the description to include a description for this other _dynamic portion_